site/docs/get-started/quickstart.md

```diff
@@ -1,61 +1,93 @@
+helm install cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --version v1.2.0 \
+    --create-namespace \
+    --set installCRDs=true \
+    --wait --timeout 120s
+``` 
 
-## Prepare for installing Mesh for Data
+## Install Hashicorp Vault and plugins
 
-1.  Obtain a local copy of Mesh for Data repository
-    ```bash
-    git clone https://github.com/ibm/the-mesh-for-data.git
-    ```
-1.  Change to the root directory of the repository
-    ```bash
-    cd the-mesh-for-data
-    ```
+[Hashicorp Vault](https://www.vaultproject.io/) and a [secrets-kubernetes-reader](https://github.com/mesh-for-data/vault-plugin-secrets-kubernetes-reader) plugin are used by Mesh for Data for credential management.
 
-## Install Mesh for Data
+Run the following to install vault and the plugin in development mode:
 
-1. Set the current namespace to `m4d-system`
+=== "Kubernetes" 
 
     ```bash
-    kubectl config set-context --current --namespace=m4d-system
+    helm repo add hashicorp https://helm.releases.hashicorp.com
+    helm install vault hashicorp/vault --version 0.9.1 --create-namespace -n m4d-system \
+        --set "server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/IBM/the-mesh-for-data/a3f951087eada4aed4b1cee9390bed5d71c35970/third_party/vault/vault-single-cluster/values.yaml \
+        --wait --timeout 120s
+    kubectl apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data/a3f951087eada4aed4b1cee9390bed5d71c35970/third_party/vault/vault-single-cluster/vault-rbac.yaml -n m4d-system
     ```
 
-1. Run the quick install script to install the control plane elements.
+=== "OpenShift"
 
     ```bash
-    ./hack/install.sh
+    helm repo add hashicorp https://helm.releases.hashicorp.com
+    helm install vault hashicorp/vault --version 0.9.1 --create-namespace -n m4d-system \
+        --set "global.openshift=true" \
+        --set "server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/IBM/the-mesh-for-data/a3f951087eada4aed4b1cee9390bed5d71c35970/third_party/vault/vault-single-cluster/values.yaml \
+        --wait --timeout 120s
+    kubectl apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data/a3f951087eada4aed4b1cee9390bed5d71c35970/third_party/vault/vault-single-cluster/vault-rbac.yaml -n m4d-system
     ```
 
-    to install on OpenShift you need to run ```WITHOUT_OPENSHIFT=false ./hack/install.sh``` instead.
+## Install control plane
 
-1. Enable the use of the [arrow flight module](https://github.com/ibm/the-mesh-for-data-flight-module)
+??? tip "Install development version"
 
+    The published Helm charts are only available for released versions. 
+    To install the `dev` version install the charts from the source code.
+    For example:
+    ```bash
+    git clone https://github.com/IBM/the-mesh-for-data.git
+    cd the-mesh-for-data
+    helm install m4d-crd charts/m4d-crd -n m4d-system --wait
+    helm install m4d charts/m4d -n m4d-system --wait
```

Comment:
When we run
helm install m4d-crd charts/m4d-crd -n m4d-system --wait
we get 
W0417 16:52:14.223158   34255 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition

When we run 
helm install m4d charts/m4d -n m4d-system --wait
we get
W0417 16:52:44.756771   34262 warnings.go:70] admissionregistration.k8s.io/v1beta1 ValidatingWebhookConfiguration is deprecated in v1.16+, unavailable in v1.22+; use admissionregistration.k8s.io/v1 ValidatingWebhookConfiguration

is this ok ?

Reply:
Hi, Yes we are currently using deprecated APIs and this is tracked in https://github.com/IBM/the-mesh-for-data/issues/501

Reply:
ok thanks